travis/maven-settings.xml

```diff
@@ -5,9 +5,9 @@
 
     <servers>
         <server>
-            <id>nexus.gadevs</id>
-            <username>${env.TRAVIS_NEXUS_USERNAME}</username>
-            <password>${env.TRAVIS_NEXUS_PASSWORD}</password>
```

Comment:
The old behaviour was for `mvn deploy` to upload build artifacts to nexus. Does this mean `mvn deploy` will not upload build artifacts to github?

Reply:
No, it is for testing.